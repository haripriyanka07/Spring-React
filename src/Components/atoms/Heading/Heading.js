import { Typography } from "@material-ui/core";

const Heading = (props) => {
    return (
        <Typography align='center' data-testid="h1tag" variant='h4'>{props.children}</Typography>
    )
}
export default Heading;

