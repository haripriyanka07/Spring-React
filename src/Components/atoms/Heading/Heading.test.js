import React from 'react';
import Heading from './Heading';
import { render, cleanup } from '@testing-library/react';
import "@testing-library/jest-dom";

afterEach(cleanup);

it("render", () => {
    const { asFragment } = render(<Heading>Heading</Heading>);
    expect(asFragment()).toMatchSnapshot();
})
it('should return the value passed as header', async() => {
    const { getByTestId } = render(<Heading>Contact</Heading>);
    expect(getByTestId("h1tag")).toHaveTextContent("Contact");
})