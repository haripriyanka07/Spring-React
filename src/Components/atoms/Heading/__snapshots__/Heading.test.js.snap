// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<DocumentFragment>
  <h4
    class="MuiTypography-root MuiTypography-h4 MuiTypography-alignCenter"
    data-testid="h1tag"
  >
    Heading
  </h4>
</DocumentFragment>
`;
